crates/communication/Cargo.toml

```diff
@@ -1,26 +1,27 @@
 [package]
-name = "communication"
-version = "0.1.0"
 edition = "2021"
-license = "GPL-3.0-only"
 homepage = "https://github.com/hulks/hulk"
+license = "GPL-3.0-only"
+name = "communication"
+version = "0.1.0"
```

Comment:
@julianschuler How do we want to sort TOMLs?

Reply:
Name, version, edition, license, homepage, so this change should be reverted.

Reply:
Fixed it, I had some plugins which were messing this up.
I found a prettier based toml plugin where the author specifically mentions rust projects ;)

```json
"[toml]": {
        "editor.defaultFormatter": "bodil.prettier-toml"
    }
```